src/backend/adapters/base-record.js

```diff
@@ -92,6 +93,7 @@ class BaseRecord {
     } catch (e) {
       if (e instanceof ValidationError) {
         this.errors = e.errors
+        this.storePayloadData()
```

Comment:
what it does? - asking because is called without the param

Reply:
its called without the params, because the save method doesn't contain parameters